### ISSUE TITLE
When viewing Silverbullet on mobile and there are more than a few buttons on the header, the page name is very truncated

### ISSUE THREAD

**Issue by dklawren** [silverbulletmd/silverbullet#1033]
Maybe it would be ideal if the buttons were converted to a hamburger menu when the screen size is small enough using CSS. Then when clicking the menu icon, the buttons drop down and can be selected. This way more of the page name would be visible on small screens.

![image](https://github.com/user-attachments/assets/9c9ec985-a199-4a8f-b12a-45ea6a82a64b)

**Comment by zefhemel** [comment]
I agree, this is bothering me too on my phone. A hamburger menu sounds like the right way to go indeed.

**Comment by jcgurango** [comment]
What if we moved these buttons to a bottom bar instead?

**Comment by zefhemel** [comment]
In practice a bottom bar is almost impossible to implement in a browser on mobile, at least on iOS. It seems very hard to reliably detect the software keyboard appearing and adapting accordingly.

**Comment by zefhemel** [comment]
https://community.silverbullet.md/t/mobile-friendly-toolbar/871?u=zef

Haven't tried this on iOS yet

**Comment by dklawren** [comment]
I tried it using Firefox's responsive design mode developer tool and selected iPhone as well as Android. It looked the same for both.

**Comment by zefhemel** [comment]
Yeah the issue is the software keyboard placement. I bet that's not taken into account in such a simulation.

**Comment by dklawren** [comment]
> Yeah the issue is the software keyboard placement. I bet that's not taken into account in such a simulation.

Not sure about iOS but on Android at least it is simple to hide the keyboard using a down arrow when i want to hit something on the bottom bar. Not sure of what would be needed to make the bar go up and down with the keyboard in both cases.

**Comment by janssen-io** [comment]
On iOS, you also have a `Done` button that closes the keyboard. :)

<img src="https://github.com/user-attachments/assets/912858fc-a2b1-40e6-b63b-95e9789697b7" height="350px" />
